src/store/loadingReducer.js

```diff
@@ -0,0 +1,30 @@
+import { type } from "@testing-library/user-event/dist/type";
+
+const initialState = {
+  loading: "false",
+};
+
+export const LOADING_POSTS = "LOADING_POSTS";
+export const LOADED_POSTS = "LOADED_POSTS";
+
+const loading = (state = initialState, payload) => {
+  switch (payload.type) {
+    case LOADING_POSTS:
+      return {
+        ...state,
+        loading: 'true',
```

Comment:
Здесь и везде юзай булев тип, а не строчки:

```suggestion
        loading: true,
```